data/lib/core/core.lua

```diff
@@ -19,3 +19,4 @@ dofile('data/lib/core/teleport.lua')
 dofile('data/lib/core/tile.lua')
 dofile('data/lib/core/vocation.lua')
 dofile('data/lib/core/quests.lua')
+dofile('data/lib/core/highscores.lua')
```

Comment:
This should be kept in alphabetical order.

`quests.lua` is out of place @MillhioreBT please move it to the alphabetical order